### ISSUE TITLE
save entries for shares

### ISSUE THREAD

**Issue by crucco37** [tinyMediaManager/tinyMediaManager#16]
Version: 2.7.3.1
Build: 2016-03-18 14:25
OS: Linux 4.2.0-34-generic
JDK: 1.8.0_74 amd64 Oracle Corporation

__What is the actual behaviour?__
i can choose network shares as folder entries for the databases
but after quitting the application their gone.

__What is the expected behaviour?__
The entry to the database on the smb share has to reappear after
the new launch of the program. the swt.jar was put into
the lib folder so i can choose smb shares as database entries.

__Steps to reproduce:__
create a database entry from an smb share, scan it, quit the application and its gone.
tried that several times. no luck.

__Additional__
Have you attached the logfile from the day it happened?

**Comment by mlaggner** [comment]
how did you add the shares? 
there are several methods in linux to mount smb/cifs (fstab, gvfs, ..). I've added all my shares from the NAS in /etc/fstab and that works like a charm (also mounting via /etc/fstab gives a much better performance over gvfs for example)

**Comment by crucco37** [comment]
Am 2016-03-23 12:48, schrieb Manuel Laggner:
> how did you add the shares?
> there are several methods in linux to mount smb/cifs (fstab, gvfs,
> ..). I've added all my shares from the NAS in /etc/fstab and that
> works like a charm (also mounting via /etc/fstab gives a much better
> performance over gvfs for example)
> 
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/tinyMediaManager/tinyMediaManager/issues/16#issuecomment-200312956
Hello,

I mounted the shares on Ubuntu 15.10 before starting tinyMM in the lib
ist stw.jar so tinyMM sees the shares as folders in the database entry 
dialog.
I added them, started the scan. After the scan i saw all the movies of 
that path.
Then I shut tinyMM and I didn't unmount the shares. After launching 
TinyMM
afterwards all the entries vanished...

**Comment by mlaggner** [comment]
could you
a) provide me the full path of your share
b) upload the config.xml after you've added the share to tmm

**Comment by crucco37** [comment]
hello manual,

here you get the path:

/run/user/1000/gvfs/smb-share:server=inferno, share=serien

after searching, I found out, that there is another config.xml in a folder with this path:

.local/share/Trash/expunged/2159072158/data

but no need for the rest. 

I found out that the folder 'data' in the tmm folder was owned by 'root'.

After creating another one owned by me and renaming the other to something
different, because I cannot delete it and I cannot reach the path with 'sudo chown'.
(another strange behavior) the config.xml shows the paths to the share so that now
the application remembers them after relaunch...

yours
_thomas


Am 26.03.2016 um 20:52 schrieb Manuel Laggner:
